docs/monitoring/metrics.md

```diff
@@ -1293,6 +1293,29 @@ Thus, in order to infer the metric identifier:
   </tbody>
 </table>
 
+#### Kinesis Connectors
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 18%">Scope</th>
+      <th class="text-left" style="width: 26%">Metrics</th>
+      <th class="text-left" style="width: 48%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th rowspan="1">Operator</th>
+      <td>millisBehindLatest</td>
+      <td>The number of milliseconds the <a>GetRecords</a> response is from the tip of the stream,
```

Comment:
Just a matter of preference here: I prefer the term "head of the stream" instead of tip.
You can ignore this if you disagree.

Reply:
I'm OK changing it. That's actually just a copy&past from Amazon docs.